android/appkotlin/src/main/java/io/o2mc/appkotlin/App.kt

```diff
@@ -22,11 +22,15 @@ class App : Application(), O2MCExceptionListener {
     o2mc.setO2MCExceptionListener(this)
   }
 
-  override fun onO2MCDispatchException(e: O2MCDispatchException?) {
-    Log.e(TAG, "onO2MCDispatchException: Some exception: handling from App.kt")
+  override fun onO2MCDispatchException(e: O2MCDispatchException) {
+    Log.e(TAG, "Handling an O2MCDispatchException from App.kt: " + e.message)
+  }
+
+  override fun onO2MCDeviceException(e: O2MCDeviceException) {
+    Log.e(TAG, "Handling an onO2MCDeviceException from App.kt: " + e.message)
   }
 
-  override fun onO2MCDeviceException(e: O2MCDeviceException?) {
-    Log.e(TAG, "onO2MCDeviceException: Some exception: handling from App.kt")
+  override fun onO2MCEndpointException(e: O2MCEndpointException) {
+    Log.e(TAG, "Handling an onO2MCEndpointException from App.kt: " + e.message)
```

Comment:
inconsistent log message; use `O2MCEndpointException` instead of `onO2MCEndpointException`.